climakitae/data/variable_descriptions.csv

```diff
@@ -0,0 +1,22 @@
+name,description,extended_description
+T2,2m Air Temperature,"Temperature of the air 2 meters above Earth's surface. This is the measure of air temperature used for most modeling applications."
+TOT_PRECIP,Precipitation (total),"Total precipitation. Computed by summing total gridscale precipitation and total cumulus precipitation."
+REL_HUMIDITY,Relative Humidity,"The amount of water vapor (i.e. moisture) in the air, relative to the amount of water vapor needed to saturate the air at the same air temperature and pressure. Expressed as a percentage."

```

Comment:
The percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.

Reply:
Fixed!